widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -1106,8 +1112,10 @@ public interface OnTitleUpdatedListener {
         void onTitleUpdated(String title);
     }
 
-    public boolean shouldShowMediaEngagementView() {
-        if (callController != null) return callController.shouldShowMediaEngagementView();
+    public boolean shouldShowMediaEngagementView(boolean isUpgradeToCall) {
```

Comment:
This is a public method. Do we need make backward compatibility for it?

Reply:
By my knowledge integrators are not using CallView separately from CallActivity at the moment but we could default it to false I guess if needed.

What do You think?

cc: @dukhovnyi